subprojects/packagefiles/libffmpegthumbnailer/meson_options.txt

```diff
@@ -0,0 +1,7 @@
+option('jpeg', type : 'feature',
+  description : 'JPEG support',
+)
```

Comment:
IMHO this probably would look a bit nicer as one line.

Reply:
Done.